chef-windows.gemspec

```diff
@@ -0,0 +1,22 @@
+gemspec = eval(IO.read(File.expand_path("../chef.gemspec", __FILE__)))
+
+gemspec.platform = Gem::Platform.new ["universal", "mingw32", nil]
```

Comment:
Will that works:
https://github.com/rubygems/rubygems/blob/master/lib/rubygems/platform.rb#L54